contracts/SolidityContractsRegistry.cdc

```diff
@@ -0,0 +1,16 @@
+access(all) contract SolidityContractsRegistry {
+
+    access(all) let contractRegistry: {String: [UInt8; 20]}
```

Comment:
Interesting idea...I wonder if something like a Cadence-native diamond pattern ([ERC-2535](https://eips.ethereum.org/EIPS/eip-2535)) would be interesting